vnext/Shared/Networking/WinRTHttpResource.cpp

```diff
@@ -167,6 +174,28 @@ fire_and_forget WinRTHttpResource::PerformSendRequest(HttpRequestMessage &&reque
   // Ensure background thread
   co_await winrt::resume_background();
 
+  // If URI handler is available, it takes over request processing.
+  if (auto uriHandler = self->m_uriHandler.lock()) {
+    auto uri = winrt::to_string(coRequest.RequestUri().ToString());
+    try {
+      if (uriHandler->Supports(uri, coReqArgs->ResponseType)) {
+        auto blob = uriHandler->Fetch(uri);
+        if (self->m_onBlobData && self->m_onRequestSuccess) {
+          self->m_onBlobData(coReqArgs->RequestId, std::move(blob));
+          self->m_onRequestSuccess(coReqArgs->RequestId);
+        }
+
+        co_return;
+      }
+    } catch (const hresult_error &e) {
+      if (self->m_onError)
+        co_return self->m_onError(coReqArgs->RequestId, Utilities::HResultToString(e));
+    } catch (const std::exception &e) {
+      if (self->m_onError)
+        co_return self->m_onError(coReqArgs->RequestId, e.what());
+    }
+  }
+
```

Comment:
Unless we implement custom URIs, this URI handler block doesn't make sense .. HTTP integration won't need it ..

Reply:
As discussed offline, not implementing custom URIs on this PR.